inception-recommendation-api/src/main/java/de/tudarmstadt/ukp/inception/recommendation/api/model/Offset.java

```diff
@@ -27,30 +27,16 @@
     private int beginCharacter = -1;
     private int endCharacter = -1;
     
-    @Deprecated
-    private int beginToken = -1;
-    
-    @Deprecated
-    private int endToken = -1;
-
-    @Deprecated
-    public Offset()
-    {
-    }
-
-    public Offset(int beginCharacter, int endCharacter, int beginToken, int endToken)
+    public Offset(int beginCharacter, int endCharacter)
```

Comment:
Should we deprecate the whole class?

Reply:
I tried deleting it, but it's used quite all over the place. This needs a bit more investigation before deprecating.